.vsts-ci/templates/test.yml

```diff
@@ -0,0 +1,8 @@
+steps:
+- bash: ./test.sh
+  displayName: 'Execute Tests'
+
+- task: PublishTestResults@2
+  inputs:
+    testResultsFormat: 'VSTest'
```

Comment:
Do you want this to fail on failed results?

Reply:
Fixed